internal/filepaths/collection_temp_dir.go

```diff
@@ -54,3 +56,38 @@ func CleanupCollectionTempDirs() {
 		}
 	}
 }
+
+// CleanupPluginTmpDirs cleans up old tmp-* directories from the plugin directory
```

Comment:
I don't think this is necessary. There is already a function `CleanupOldTmpDirs` in pipe-fittings which does the same thing. We should use that function - https://github.com/turbot/pipe-fittings/blob/develop/plugin/cleanup.go#L13

The same is implemented in Steampipe - https://github.com/turbot/steampipe/blob/develop/cmd/plugin.go#L78-L82